modules/ROOT/pages/deployment/services/s-list/audit.adoc

```diff
@@ -30,6 +28,8 @@ Example json:
 
 The audit service is not started automatically when running as single binary started via `ocis server` or when running as docker container and must be started and stopped manually on demand.
 
+Per default, it will be logged to standard out, but can also be configured to a file output. Note that when a file output is used, it is not part of the standard log file but a separate one.
```

Comment:
```suggestion
Per default, it will be logged to standard out, but can also be configured to write into a file. Note that when a file output is used, it is not part of the standard log file but a separate one.
```